pint/testsuite/test_numpy.py

```diff
@@ -154,34 +521,82 @@ def test_clip(self):
         self.assertRaises(ValueError, self.q.clip, self.ureg.J)
         self.assertRaises(ValueError, self.q.clip, 1)
 
+    @helpers.requires_array_function_protocol()
+    def test_clip_numpy_func(self):
+        self.assertQuantityEqual(np.clip(self.q, 150 * self.ureg.cm, None), [[1.5, 2], [3, 4]] * self.ureg.m)
+
     def test_round(self):
         q = [1, 1.33, 5.67, 22] * self.ureg.m
         self.assertQuantityEqual(q.round(0), [1, 1, 6, 22] * self.ureg.m)
         self.assertQuantityEqual(q.round(-1), [0, 0, 10, 20] * self.ureg.m)
         self.assertQuantityEqual(q.round(1), [1, 1.3, 5.7, 22] * self.ureg.m)
 
+    @helpers.requires_array_function_protocol()
+    def test_round_numpy_func(self):
+        self.assertQuantityEqual(np.around(1.0275 * self.ureg.m, decimals=2), 1.03 * self.ureg.m)
+        self.assertQuantityEqual(np.round_(1.0275 * self.ureg.m, decimals=2), 1.03 * self.ureg.m)
+
     def test_trace(self):
         self.assertEqual(self.q.trace(), (1+4) * self.ureg.m)
 
     def test_cumsum(self):
         self.assertQuantityEqual(self.q.cumsum(), [1, 3, 6, 10] * self.ureg.m)
 
+    @helpers.requires_array_function_protocol()
+    def test_cumsum_numpy_func(self):
+        self.assertQuantityEqual(np.cumsum(self.q, axis=0), [[1, 2], [4, 6]] * self.ureg.m)
+
+    @helpers.requires_array_function_protocol()
+    def test_nancumsum_numpy_func(self):
+        self.assertQuantityEqual(np.nancumsum(self.q_nan, axis=0), [[1, 2], [4, 2]] * self.ureg.m)
+
     def test_mean(self):
         self.assertEqual(self.q.mean(), 2.5 * self.ureg.m)
 
+    @helpers.requires_array_function_protocol()
+    def test_mean_numpy_func(self):
+        self.assertEqual(np.mean(self.q), 2.5 * self.ureg.m)
+        self.assertEqual(np.mean(q_temperature), self.Q_(2.5, self.ureg.m))
```

Comment:
I guess that's the last error not related to missing implementations, except from the one related to `np.power`?
```suggestion
        self.assertEqual(np.mean(self.q_temperature), self.Q_(2.5, self.ureg.degC))
```